custom/icds_reports/ucr/data_sources/dashboard/delivery_forms.json

```diff
@@ -242,6 +242,39 @@
           "type": "property_path",
           "property_path": ["form", "next_due"]
         }
+      },
+      {
+        "column_id": "num_children_del",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "property_path",
+          "property_path": ["form", "num_children_del"]
+        }
+      },
+      {
+        "column_id": "still_birth",
```

Comment:
nit: is there a particular reason to use `still_birth` over `still_live_birth` ?

Reply:
I was thinking of it as a boolean and so the name made more sense. But if we think it's better to treat like an enum then I can change it.

Reply:
I wasn't thinking of the change in column_id as a difference between boolean vs enum. I was suggesting it since that's what the question name in the app was. We still have a lot of columns where they aren't the same, but have mostly been using the same names between app and column name when possible.

Reply:
Yeah that makes sense and is probably a good convention to keep. I think what I didn't like about the current name and a boolean is that it would not be obvious which value of `still_live_birth` should be `0` versus `1` whereas with the changed name it's much more intuitive. But I think that's a small/surmountable issue.

Reply:
https://github.com/dimagi/commcare-hq/pull/25720/commits/967f17f5cf9093e598d867c5f9e76b1f43edc732